### PR TITLE
feat:edit stories to show full code and add test code

### PR DESCRIPTION
**Pull request by wjdwl002**
#251

### CODE REVIEW

File: src/CodeMockup/CodeMockup.stories.tsx [hunk 1]
```diff
@@ -14,55 +14,57 @@ const meta: Meta = {
 
 export default meta
 
-const Template: Story<CodeMockupProps> = (args) => {
-  return <CodeMockup {...args} />
+export const Default: Story<CodeMockupProps> = (args) => {
+  return (<CodeMockup {...args} >
```
Comment (benjitrosch) [inline src/CodeMockup/CodeMockup.stories.tsx]: For style consistency, can you add a newline inbetween the parenthesis, so this becomes:
```js
return (
     <CodeMockup>
     ...
     </CodeMockup>
)
```

File: src/CodeMockup/CodeMockup.test.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    it('should have no line prefix', () => {
+        render(<CodeMockup>
+            <CodeMockup.Line dataPrefix={false} data-testid="code-line"></CodeMockup.Line>
+        </CodeMockup>)
+        
+        expect(getComputedStyle(screen.getByTestId('code-line'), ':before').getPropertyValue("content")).toEqual("")
+    })  
+
+    it('should ')
```
Comment (benjitrosch) [inline src/CodeMockup/CodeMockup.test.tsx]: Looks like an incomplete test?

**Review by benjitrosch — CHANGES_REQUESTED** [review]
This looks good, thanks for making this PR @wjdwl002! Just needs a couple very small fixes before being merged 👍

File: src/CodeMockup/CodeMockup.test.tsx (hunk 2 above)
Comment (wjdwl002) [reply src/CodeMockup/CodeMockup.test.tsx]: Oh no it was my mistake 😅 I'll fix and recommit it !

File: src/CodeMockup/CodeMockup.stories.tsx (hunk 1 above)
Comment (wjdwl002) [reply src/CodeMockup/CodeMockup.stories.tsx]: fixed it on [9a48907](https://github.com/daisyui/react-daisyui/pull/252/commits/9a489078979eacac01d9e6223870bd50ffb80fdf) 
not very sure this is what it meant to be tho 🤔

**Review by benjitrosch — APPROVED** [review]

**Pull request merged**